mypy/types.py

```diff
@@ -272,9 +272,12 @@ def serialize(self) -> JsonDict:
 class AnyType(Type):
     """The type 'Any'."""
 
-    def __init__(self, implicit: bool = False, line: int = -1, column: int = -1) -> None:
+    def __init__(self, implicit: bool = False, from_silent_import: bool = False,
+                 line: int = -1, column: int = -1) -> None:
         super().__init__(line, column)
-        self.implicit = implicit
+        self.implicit = implicit  # Was this Any type was inferred without a type annotation?
```

Comment:
Nitpick: probably better to just put this comment on the preceding line for consistency with the next member variable initalization.